packages/CloseButton/styles.js

```diff
@@ -0,0 +1,44 @@
+import styled, { css, keyframes } from '@xstyled/styled-components'
+import { th } from '@xstyled/system'
+import { Button } from '@welcome-ui/button'
+import { system } from '@welcome-ui/system'
+import { hexToRGBA } from '@welcome-ui/utils'
+
+const fade = keyframes`
```

Comment:
I think is useless, it was only on ClearButton to add a good effect on button apparition.